googler

```diff
@@ -1175,13 +1206,22 @@ class GoogleParser(html.parser.HTMLParser):
             self.set_handlers_to('result')
             return 'result'
 
+        # Autocorrect
+        if 'spell_orig' in self.classes(attrs) and attrs.get('id') not in {'sifm', 'sifl'}:
```

Comment:
Can we put this inside a tag check somewhere to avoid `'spell_orig' in self.classes(attrs)` every time?